packages/fluent/scss/mediaplayer/_theme.scss

```diff
@@ -0,0 +1,30 @@
+@use "_variables.scss" as *;
+@use "../core/mixins" as *;
+
+@mixin kendo-mediaplayer--theme() {
+
+    .k-mediaplayer {
+        @include fill(
+            var( --kendo-mediaplayer-text, #{$kendo-mediaplayer-text} ),
+            var( --kendo-mediaplayer-bg, #{$kendo-mediaplayer-bg} ),
+            var( --kendo-mediaplayer-border, #{$kendo-mediaplayer-border} )
+        );
+    }
+
+
+    .k-mediaplayer-overlay {
+        background-color: var( --kendo-mediaplayer-overlay-bg, #{$kendo-mediaplayer-overlay-bg} );
+    }
+
+
+    .k-mediaplayer-titlebar {
+        color: var( --kendo-mediaplayer-titlebar-text, #{$kendo-mediaplayer-titlebar-text} );
+        background-color: var( --kendo-mediaplayer-titlebar-bg, #{$kendo-mediaplayer-titlebar-bg} );
+        text-shadow: 0 0 2px rgba( var( --kendo-mediaplayer-text, #{$kendo-mediaplayer-text} ), .5 );
```

Comment:
rgba on css variables does not work. Use a separate variable and point to a color value from the palette (no opacity).

Reply:
Looks like the titlebar is not supposed to have text shadow so I had it removed. Thanks!